src/commcare_cloud/fab/operations/release.py

```diff
@@ -460,17 +459,6 @@ def copy():
     return copy
 
 
-@parallel
-@roles(ROLES_FORMPLAYER)
-def copy_formplayer_properties():
```

Comment:
this was part of the lazy migration for moving formplayer to it's own release location and is no longer required